src/templates/React InstantSearch Native/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(api) {
```

Comment:
When releasing on npm, we need to check that this file is published correctly. This wasn't the case with `.babelrc` (hence the name `.babelrc.template`).